Makefile

```diff
@@ -5,8 +5,9 @@ PREFIX ?= /usr
 BINDIR ?= ${PREFIX}/bin
 UID=$(shell id -u)
 GOCACHEDIR=$(shell which go >/dev/null 2>&1 && go env GOCACHE || echo "$(HOME)/.cache/go-build")
+GO111MODULE=on
```

Comment:
I think the issue is that the line above does not set the env variable `$GO111MODULE` expected.

So if I manually set `GO111MODULE=on` for the go build command, it works fine.
On the other hand, if I run it without `GO111MODULE=on`, it sometimes fails, because it tries to look for the cni plugins repo under my GOPATH. If my local cni plugins repo is based on 0.7, it works fine. If not, it fails.
That's not exactly what users would expect to happen.

Reply:
Ah, that's my missing knowledge regarding `Makefile`. Should I add `export GO111MODULE=on` to `all` and `update-vendor` then?

Reply:
Yes, you can do so.
Or you can simply add `GO111MODULE=on` only for go build commands.
If it works, I don't care much.